src/components/BreedsList.vue

```diff
@@ -114,7 +114,14 @@ export default {
   },
   computed: {
     displayedBreeds() {
-      return this.searchList.length > 0 ? this.searchList : this.breedsList;
+      if (this.searchList.length > 0) {
+        return this.searchList;
+      } else if (this.searchList.length === 0 && this.searchQuery !== "") {
+        this.searchList = [];
+        return this.searchList;
+      } else {
+        return this.breedsList;
+      }
```

Comment:
Če ne najde zadetka pri iskanju ti izpiše samo to` Your searches for "{{ this.searchQuery }}" did not have any matches.
          Try different keywords.` in ne prikaže vseh breedsov